constrained-voucher.mkd

```diff
@@ -1219,6 +1219,14 @@ The following security considerations have led to this choice of scope:
 
 # IANA Considerations
 
+## GRASP Discovery Registry
+
+IANA is asked to extend the registration of the "AN\_Proxy" (without quotes) in the "GRASP Objective Names" table in the Grasp Parameter registry.
+This document should also be cited for the new protocol value UDP defined in {{grasppledgediscovery}}.
```

Comment:
```suggestion
This document should also be cited for this existing registration, because {{grasppledgediscovery}} defines the new protocol value IPPROTO_UDP for the objective.
```

Reply:
Tried to clarify this. Since the registry doesn't list value "IPPROTO_TCP" or "IPPROTO_UDP" and so on, we probably just want to cite our document as the reference here.